packages/openapi3/src/xml-module.ts

```diff
@@ -136,8 +136,12 @@ export async function resolveXmlModule(): Promise<XmlModule | undefined> {
           scalarSchema.xml = { name: propXmlName };
           refSchema.items = scalarSchema;
         } else {
+          const items = new ArrayBuilder();
+          for (const item of Array.from([refSchema.items])) {
```

Comment:
this feels like this just create a single value array and just iterate over it to add right back up